test/parallel/test-cluster-disconnect.js

```diff
@@ -1,41 +1,41 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var cluster = require('cluster');
-var net = require('net');
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const net = require('net');
 
 if (cluster.isWorker) {
-  net.createServer(function(socket) {
+  net.createServer((socket) => {
     socket.end('echo');
   }).listen(common.PORT, '127.0.0.1');
 
-  net.createServer(function(socket) {
+  net.createServer((socket) => {
     socket.end('echo');
   }).listen(common.PORT + 1, '127.0.0.1');
 
 } else if (cluster.isMaster) {
   var servers = 2;
 
   // test a single TCP server
-  var testConnection = function(port, cb) {
-    var socket = net.connect(port, '127.0.0.1', function() {
+  const testConnection = function(port, cb) {
+    var socket = net.connect(port, '127.0.0.1', () => {
       // buffer result
       var result = '';
-      socket.on('data', function(chunk) { result += chunk; });
+      socket.on('data', common.mustCall((chunk) => { result += chunk; }));
```

Comment:
I've seen test failures in the past because multiple `data` event calls were not considered, but we can wait for the test to fail before making any change.